guides/common/modules/proc_preparing-for-using-external-databases.adoc

```diff
@@ -0,0 +1,27 @@
+[id="Preparing_for_using_external_databases_{context}"]
+= Optional: Preparing {Project} for using external databases
+
+By default, the {Project} installation process includes installing a PostgreSQL database on the same host as {ProjectServer}.
+However, in certain {Project} deployments, using external databases instead of the default local databases can help with the server load.
+
+You can set up your {Project} deployment to use an external PostgreSQL database during the installation of {Project}. 
+You can also migrate to an external PostgreSQL database on an existing {Project} deployment.
+
+.Procedure
+
```

Comment:
```suggestion
```
This applies to the other discreet headings as well. A blank line is unnecessary.